src/api-reference/locate/userLocations.markdown

```diff
@@ -116,7 +116,7 @@ This information will be provided to you along with your client ID and secret.
 | description | 100 | String |
 
 ### Transaction
-Itineraries can be added or cancelled. This section allows you to indicate whether this is an addition of a new itinerary or cancellation of an existing itinerary.
+Itineraries can be added or cancelled. This section allows you to indicate whether this is an addition of a new itinerary or cancellation of an existing itinerary.A Cancel request for this scenario is a partial cancellation or update to an existing transaction so they simply need to resend that transaction request without that cancelled segment and our service removes the previous transaction details completely and saves the latest transaction details
```

Comment:
A cancel request is for canceling the itinerary completely in our system and   if there are any updates to the existing itineraries then they need to be resent using the same booking or transaction id.